app/services/converters/order_detail_to_journal_rows.rb

```diff
@@ -0,0 +1,23 @@
+module Converters
+  class OrderDetailToJournalRowAttributes
+
+    attr_accessor :journal, :order_detail, :total
+
+    def initialize(journal, order_detail, options = {})
+      @journal = journal
+      @order_detail = order_detail
+      @total = options[:total] || order_detail.total
+    end
+
+    def convert
```

Comment:
I suspect for NU we'll need to write a modified version of this converter that handles the additional journal fields there. I'd like to make sure we have a functional PR on NU before we merge this into open/master.